codespell_lib/data/dictionary.txt

```diff
@@ -16755,7 +16755,9 @@ spacific->specific
 spacifics->specifics
 spacified->specified
 spacifies->specifies
+spageti->spaghetti
 spagetti->spaghetti
```

Comment:
You may as well add `spagheti->spaghetti` while you're here.

Reply:
Done